go.mod

```diff
@@ -1,21 +1,31 @@
 module rke-tools
 
-go 1.16
+go 1.19
```

Comment:
Golang is 1.20 but this is 1.19

Reply:
Should we update the Go version contract to `1.20` instead? The upgrade to `1.19` was to ensure we were on the minimum supported version. The version of the compiler does not need to be aligned with this, but I am happy to bump this if you think that's important.